### PR TITLE
Fix miscellaneous code errors.

### PR DESCRIPTION
**Pull request by tech4him1**
**- Summary**

While working on #952, I noticed that there were a few actual code errors that the linter was picking up. I think they should be fixed sooner, rather than waiting until we can merge the entire linted/prettified code.

**- Test plan**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: src/backends/git-gateway/AuthenticationPage.js [hunk 1]
```diff
@@ -41,7 +41,8 @@ export default class AuthenticationPage extends React.Component {
   }
 
   handleIdentity = () => {
-    if (window.netlifyIdentity.currentUser()) {
+    const user = window.netlifyIdentity.currentUser();
+    if (user) {
       this.props.onLogin(user);
```
Comment (tech4him1) [inline src/backends/git-gateway/AuthenticationPage.js]: This may be the fix for https://github.com/netlify/netlify-cms/issues/1231.

**Comment by verythorough** [comment]
Deploy preview for *cms-demo* ready!

Built with commit 13455343420ea128e7889d65a6415e758d197a49

https://deploy-preview-1251--cms-demo.netlify.com

**Comment by verythorough** [comment]
Deploy preview for *netlify-cms-www* ready!

Built with commit 13455343420ea128e7889d65a6415e758d197a49

https://deploy-preview-1251--netlify-cms-www.netlify.com

**Comment by ekoeryanto** [comment]
Great!
I hope it fixes #1231 

> Haha, glad i'm not reading all the the code yet 😆

**Pull request merged**